enterprise_access/apps/api/v1/views/subsidy_access_policy.py

```diff
@@ -966,25 +1024,58 @@ def get_group_member_data_with_aggregates(self, request, uuid):
             policy.uuid
         )
 
+        # If `sort_by_enrollment_count` is true, then we need to fetch all the group members records and do the sorting
+        # ourselves since platform is unaware of enrollment data numbers.
+        request_page = None
+        request_traverse_pagination = None
+        if sort_by_enrollment_count := (sort_by == SORT_BY_ENROLLMENT_COUNT):
+            # Mock out data sent to platform to fetch the data we need to sort
+            # ie set `traverse_pagination` to True
+            sort_by = None
+            request_traverse_pagination = traverse_pagination
+            traverse_pagination = True
+            request_page = page
+            page = None
```

Comment:
There should really be a warning, probably in the request serializer help text for the `sort_by` field, that setting `sort_by_enrollment_count` will cause the incoming request to wait for _all_ pages of data to be fetched from the edx-enterprise/LMS endpoint.